CONTRIBUTING.md

```diff
@@ -44,10 +44,13 @@ To get started clone the repo and get the web application started.
  5. Point your browser to [http://localhost:4242](http://localhost:4242).
  6. If port `4242` is taken, then you can run the web app on a different port: `FX_PROFILER_PORT=1234 yarn start`
 
-Other [webpack](https://webpack.js.org/configuration/) and [webpack server](https://webpack.js.org/configuration/dev-server/) options can be set in a `webpack.local-config.js` file at the repo root. For example, if you want the server to automatically open the home page, put in there the following code:
+Other [webpack](https://webpack.js.org/configuration/) and [webpack server](https://webpack.js.org/configuration/dev-server/) options can be set in a `webpack.local-config.js` file at the repo root. For example, if you want a persistent build cache and the server to automatically open the home page, put in there the following code:
 
 ```js
 module.exports = function (config, serverConfig) {
+  config.cache = {
+    type: 'filesystem',
+  };
```

Comment:
Mmm I wonder if we should have this by default in our config actually. It looks like I considered doing this because I have an open tab on this configuration.
It would be good to get your feedback after a few days using this.

Reply:
I currently do plan to find out whether server config changes force a full rebuild, but what is next isn't yet set in stone.

Reply:
Modifying `serverConfig.static` roughly triples the webpack build time lol

Reply:
Modifying `serverConfig.static` roughly triples the cached webpack build time lol